sbndcode/OpDetSim/TriggerEmulationService.cc

```diff
@@ -17,7 +17,7 @@ namespace calib {
     : fMonWidth(pset.get<int>("MonWidth", 12)),
       fTotalCAENBoards(pset.get<int>("TotalCAENBoards", 8)),
       PMTPerBoard(pset.get<int>("PMTPerBoard", 15)),
-      Baseline(pset.get<int>("Baseline", 14250)),
+      Baseline(pset.get<int>("Baseline", 14257)),
```

Comment:
Same here, we want to keep run1 baseline value by now. In general, it is good practice not giving default values in the constructor so we make sure everything is set up from the fhicl.
```suggestion
      Baseline(pset.get<int>("Baseline")),
```